yarn.lock

```diff
@@ -2300,6 +2300,28 @@
     lodash "^4.17.14"
     web3-utils "1.0.0-beta.37"
 
+"@celo/utils@0.1.4", "@celo/utils@^0.1.0":
```

Comment:
it seems there's somebody still using 0.1.0. I wonder who he is

Reply:
I was looking into this, and I seems like old versions of all of our packages are in here. For example `contractkit` still has a reference to its v0.1.5 in `yarn.lock` I can't find any  reference to `@celo/utils@0.1.0` anywhere else.
https://github.com/celo-org/celo-monorepo/blob/cbdd336341a5d63bd06d232e0056f71320b34252/yarn.lock#L2234-L2251